### ISSUE TITLE
msodde UnicodeEncodeError

### ISSUE THREAD

**Issue by securitykitten** [decalage2/oletools#267]
Hello, I'm encountering an UnicodeEncodeError when running the latest version of msodde.

```
Traceback (most recent call last):
  File "/usr/local/bin/msodde", line 9, in <module>
    load_entry_point('oletools==0.52.2', 'console_scripts', 'msodde')()
  File "/usr/local/lib/python2.7/dist-packages/oletools/msodde.py", line 1018, in main
    text = process_file(args.filepath, args.field_filter_mode)
  File "/usr/local/lib/python2.7/dist-packages/oletools/msodde.py", line 976, in process_file
    return process_docx(filepath, field_filter_mode)
  File "/usr/local/lib/python2.7/dist-packages/oletools/msodde.py", line 611, in process_docx
    if not field_is_blacklisted(field.strip())]
  File "/usr/local/lib/python2.7/dist-packages/oletools/msodde.py", line 662, in field_is_blacklisted
    .format(contents, FIELD_BLACKLIST[index]))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xa0' in position 14: ordinal not in range(128)
```

A document that can reproduce this bug can be found here -> https://drive.google.com/open?id=11VHx4T4Pw9stgWxvk5k2HdgWJx4cavCG

password: oletools

**Comment by balin77** [comment]
Hi, thank you very much for your code. Its very useful and saved my but. But now I am encountering similar troubles. Im using olevba3 --code. This works fine up to some point. then I get this:

```
ERROR    Unhandled exception in main: Error processing file 
C:\Users\...Filename.xlsm ('charmap' codec can't encode 
character '\u03f0' in position 582: character maps to <undefined>)
Traceback (most recent call last):
File "c:\users\.....\programs\python\python36-32\lib\site- 
packages\oletools\olevba3.py", line 3221, in process_file
print(vba_code_filtered)
File "c:\users\.....\appdata\local\programs\python\python36-32\lib\encodings\cp1252.py", line 19, in encode
 return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u03f0' in position 582: character maps to <undefined>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\.....appdata\local\programs\python\python36-32\lib\site-packages\oletools\olevba3.py", line 3607, in main
    deobfuscate=options.deobfuscate)
  File "c:\users\......\appdata\local\programs\python\python36-32\lib\site-packages\oletools\olevba3.py", line 3241, in process_file
    raise ProcessingError(self.filename, exc)
oletools.olevba3.ProcessingError: Error processing file 
C:\Users\...Filename.xlsm ('charmap' codec can't encode character '\u03f0' in position 582: character maps to <undefined>)
```

**Comment by decalage2** [comment]
Thanks, would it be possible to share the file that triggers the error?

**Comment by balin77** [comment]
Im afraid, I actually cant. Its an Excel of my company with quite sensible data in it. Im really sorry. But Im happy to test any changes you might make.

**Comment by decalage2** [comment]
@balin77 this is a different bug, so I moved it to #323.

**Comment by aleixsb** [comment]
Same error here! I can't share the document either..
msodde 0.53 - http://decalage.info/python/oletools
THIS IS WORK IN PROGRESS - Check updates regularly!
Please report any issue at https://github.com/decalage2/oletools/issues

Opening file: test.doc
Traceback (most recent call last):
  File "/usr/local/bin/msodde", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/oletools/msodde.py", line 1021, in main
    text = process_file(args.filepath, args.field_filter_mode)
  File "/usr/local/lib/python2.7/dist-packages/oletools/msodde.py", line 951, in process_file
    return process_doc(filepath)
  File "/usr/local/lib/python2.7/dist-packages/oletools/msodde.py", line 524, in process_doc
    .format(direntry.entry_type)))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xc3' in position 0: ordinal not in range(128)

**Comment by decalage2** [comment]
Fixed by PR #469
